<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Saleae</class>
 <widget class="QMainWindow" name="Saleae">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>482</width>
    <height>368</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Saleae, Version 1.0 June 25, 2015</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="pbConnect">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>461</width>
      <height>26</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Press this button to connect to the Saleae Logic application. Makre sure the Logic application is running. This is the first step in using this interface application.</string>
    </property>
    <property name="text">
     <string>Connect to Saleae device, make sure Logic app is running</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboDevices">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>35</y>
      <width>386</width>
      <height>21</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Select your device from this list of devices.</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Devices</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>241</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Number of analog channels to record</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboNumChannels">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>70</y>
      <width>196</width>
      <height>21</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Select how many channels you wish to record.</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="gbTrigger">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>100</y>
      <width>161</width>
      <height>131</height>
     </rect>
    </property>
    <property name="title">
     <string>Trigger options</string>
    </property>
    <widget class="QRadioButton" name="rbHigh">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>61</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>high</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="rbLow">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>51</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>low</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="rbNegedge">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>60</y>
       <width>81</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>negedge</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QRadioButton" name="rbPosedge">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>90</y>
       <width>81</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>posedge</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Trigger channel</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="leTriggerChan">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>30</y>
       <width>31</width>
       <height>21</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>120</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Pre trigger</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>160</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Sample rate</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>200</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Number of samples</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboSampleRates">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>150</y>
      <width>176</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="leNumberSamples">
    <property name="geometry">
     <rect>
      <x>335</x>
      <y>195</y>
      <width>126</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>100000</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lePreTrigger">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>110</y>
      <width>171</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="leFileName">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>245</y>
      <width>321</width>
      <height>21</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Enter file name or use the file selection dialog button to your right. If the file ends with a number then this application will automatically advance the number after a successful acquisition.</string>
    </property>
    <property name="text">
     <string>testfile.0001</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>250</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>File name</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbFileSelect">
    <property name="geometry">
     <rect>
      <x>415</x>
      <y>246</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Press this button to see a file selection dialog box.</string>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbAcquire">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>281</y>
      <width>211</width>
      <height>26</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Acquire from the Logic device.</string>
    </property>
    <property name="text">
     <string>Acquire</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="chkRepeat">
    <property name="geometry">
     <rect>
      <x>235</x>
      <y>280</y>
      <width>87</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Repeat</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>482</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
    <addaction name="separator"/>
   </widget>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
